discovery_child_development/pipeline/patents/fetch_patents.py

```diff
@@ -0,0 +1,42 @@
+"""
+Script to fetch patents from Google Patents BigQuery database.
+
+NB: Running this will incur costs (scanning 1TB ~ £6)
+If you're changing the query, first experiment with dry_run() to
+get an estimate of the query size, or use BigQuery's query explorer.
+
+Usage:
+
+python discovery_child_development/pipeline/patents/fetch_patents.py
+"""
+from discovery_child_development.utils import google_utils, keywords as kw
+from discovery_child_development import PROJECT_DIR
+
+KEYWORD_FILE = PROJECT_DIR / "discovery_child_development/config/patents/keywords.txt"
+PATENT_PATH = "data/patents/"
+
+if __name__ == "__main__":
+    keywords = kw.get_keywords(KEYWORD_FILE)
+    # Extend the set of keywords
+    keywords = kw.deduplicate_keywords(
+        keywords
+        + kw.replace_word(keywords, "child", "infant")
+        + kw.replace_word(keywords, "child", "baby")
+        + kw.replace_word(keywords, "child", "toddler")
+    )
+    # Save the final list of keywords
+    query_keywords_path = KEYWORD_FILE.parent / "keywords_query.txt"
+    kw.save_keywords(keywords, query_keywords_path)
+    query = google_utils.create_patents_query(keywords)
+    # Run the query
+    client = google_utils.create_client()
+    google_utils.dry_run(client, query)
```

Comment:
Is it worth either adding a `time.sleep()` or requiring a user input here? I am thinking of how to use `dry_run()` so that if from the logs the query looked much bigger than you were expecting, you would have the option to kill the process 😅

Reply:
Nice idea - I'm thinking as this is a pipeline script, the user has already checked that the query is OK and implemented it in the pipeline. The dry run here is hence just for reference. But you're right, we could add a user prompt - will look into this